.teamcity/pom.xml

```diff
@@ -9,7 +9,7 @@
 
     <name>Develocity API adapters TeamCity Config DSL</name>
     <groupId>com.gradle</groupId>
-    <artifactId>develocity-api-adapters</artifactId>
```

Comment:
I'm just not sure about this tbh, but afaik there's no `develocity-api-adapters` repo or artifact, so I think it's fine.